### ISSUE TITLE
The results that applied blur are different. <composited video(ffmpeg) vs browser(jassub)>

### ISSUE THREAD

**Issue by lowkey-heonyoung** [ThaUnknown/jassub#45]
I've set blur to **3** to `어때요` text, then do composite video through ffmpeg library with ASS subtitle data.
However, the result is different than I expected like below. Especially, when I set the blur, the result has an issue.
I'll appreciate it if someone could help me.

| composited video(ffmpeg)    | browser(jassub) |
| -------- | ------- |
| <img width="800" alt="composited(ffmpeg)" src="https://github.com/user-attachments/assets/969af216-1d7c-4055-975d-37bcb695c1de"> | <img width="800" alt="preview(jassub)" src="https://github.com/user-attachments/assets/7918b5bd-a505-43e6-9d61-78e3a3ec9196">|

```
[Script Info]
ScriptType: v4.00+
PlayResX: 384
PlayResY: 288
ScaledBorderAndShadow: yes
YCbCr Matrix: None
WrapStyle: 0
Collisions: Reverse

[V4+ Styles]
Format: Name, Fontname, Fontsize, PrimaryColour, SecondaryColour, OutlineColour, BackColour, Bold, Italic, Underline, StrikeOut, ScaleX, ScaleY, Spacing, Angle, BorderStyle, Outline, Shadow, Alignment, MarginL, MarginR, MarginV, Encoding
Style: Title,Pretendard ExtraBold,28,&H00ffffff,&H00ffffff,&H00000000,&H00000000,0,0,0,0,100,100,,,1,0.5,1.0,2,10,10,161,
Style: Title-box,Pretendard ExtraBold,28,&H00000000,&H00000000,&H00000000,&Hff000000,0,0,0,0,100,100,,,3,1.0,,2,10,10,161,
Style: Subtitle,KOTRA HOPE,44,&H001c1cd3,&H00ffffff,&H00000000,&Hf0000000,0,0,0,0,100,100,,,1,1.0,1.0,2,10,10,77,
Style: Custom1,NanumSquare Neo ExtraBold,44,&H001c1cd3,&H00ffffff,&H00000000,&H0022f6ff,0,0,0,0,100,100,,,1,2.0,2.4,2,10,10,77,
Style: Custom2,NanumSquare Neo ExtraBold,22,&H001c1cd3,&H00ffffff,&H00000000,&Hf0000000,0,0,0,0,100,100,,,1,1.0,1.0,2,10,10,77,

[Events]
Format: Layer, Start, End, Style, Name, MarginL, MarginR, MarginV, Effect, Text
Dialogue: 11,0:04:27.00,0:04:30.00,Title,,0,0,0,,{\blur6}자 오늘 주제는 뭐죠?
Dialogue: 10,0:04:27.00,0:04:30.00,Title-box,,0,0,0,,자 오늘 주제는 뭐죠?
Dialogue: 1,0:04:27.18,0:04:28.20,Subtitle,,0,0,0,,{\rCustom2}아니{\r} {\rCustom1}{\blur3}어때요?{\r}
```

**Comment by ThaUnknown** [comment]
if you're using [dropAllBlur](https://github.com/ThaUnknown/jassub/blob/main/src/jassub.js#L47), disable it, if not ask libass, this is merely a wrapper for libass, it doesn't perform any of the rasterization

**Comment by lowkey-heonyoung** [comment]
@ThaUnknown Thanks a lot for your advice. I'll contact to libass.

**Comment by lowkey-heonyoung** [comment]
@ThaUnknown
Hi, I've received an answer on this issue from **libass** that it can be related to not calling `ass_set_storage_size`.
Could you take a look of this?

> Looks like jassub isn't calling `ass_set_storage_size`. Blur has this kind of scaling behavior when that isn't called; this is expected. Please report downstream.
https://github.com/libass/libass/issues/810#issuecomment-2262276155

**Comment by ThaUnknown** [comment]
this is possible, i'll debug this sometime this month

**Comment by ThaUnknown** [comment]
@lowkey-heonyoung you can try the version provided in the git commit artifacts on the latest commit, I haven't published it on NPM yet

**Comment by lowkey-heonyoung** [comment]
@ThaUnknown I've tried to build the latest commit, but it's quite hard to build.. Can you share the build result so that I can try in the local environment.

**Comment by ThaUnknown** [comment]
as I said it's in the commit build artifacts

**Comment by ThaUnknown** [comment]
https://github.com/ThaUnknown/jassub/actions/runs/10255187904

**Comment by lowkey-heonyoung** [comment]
@ThaUnknown The blur is showing appropriately when running the new version you shared. Thanks a lot.

**Comment by ThaUnknown** [comment]
published as 1.7.17
